src/Uno.UI/Controls/RootViewController.iOS.cs

```diff
@@ -60,6 +60,12 @@ private void Initialize()
 			UIApplication.Notifications
 				.ObserveWillResignActive((sender, args) =>
 					VisualTreeHelper.CloseLightDismissPopups(WinUICoreServices.Instance.ContentRootCoordinator.CoreWindowContentRoot.XamlRoot));
+
+			// iOS 17+ only
+			if (UIDevice.CurrentDevice.CheckSystemVersion(17, 0))
```

Comment:
Won't we double the calls to RefreshSystemTheme() ?

Reply:
Technically yes, but it is not a problem - the `RefereshSystemTheme` is lightweight and if the theme did not actually change, it is close to no-op